.circleci/config.yml

```diff
@@ -44,20 +47,36 @@ references:
 jobs:
 
   # x64
-  manylinux-x64_cp27-cp27mu:
-    <<: *x64_build_job
   manylinux-x64_cp37-cp37m_upload-sdist:
     <<: *x64_build_job
 
   # x86
-  manylinux-x86_cp27-cp27mu:
-    <<: *x86_build_job
   manylinux-x86_cp37-cp37m:
     <<: *x86_build_job
 
+  # python 2.7 tests
+  manylinux-x64_cp27-cp27mu_tests-only:
+    <<: *x64_build_job
+
+  check-dist:
```

Comment:
Since this not only "check" the distribution, but also build it. What about changing it to `build-and-test-dist`

Reply:
Not sure I'm following what you're saying.
`check-dist` uses the same semantics as `deploy-master` or `deploy-release`.
`check-dist` doesn't build anything on its own but requires all artefacts built by `manylinux-x64_cp37-cp37m_upload-sdist` & `manylinux-x86_cp37-cp37m`. (The additional requires on `manylinux-x64_cp27-cp27mu_tests-only` is to make sure all tests are done before checking/deploying)

Reply:
Good point. This was on oversight on my side.